src/Common/LoggingFormatStringHelpers.h

```diff
@@ -113,12 +123,37 @@ template <typename T> constexpr std::string_view tryGetStaticFormatString(T && x
     }
 }
 
+void tryGetFormattedArgs(std::vector<std::string>&, bool) { }
+
+template <typename T> void tryGetFormattedArgs(std::vector<std::string>& out, bool ignoreFirst, T && x)
+{
+    if (!ignoreFirst)
```

Comment:
What's the point of `ignoreFirst`? Should we avoid calling this function instead of passing `ignoreFirst = true`? 
Also, function arguments must be in snake_case

Reply:
I had some problems with resolving a macro to a function call when there are no formatting arguments when I excluded the format string, but I can double-check